From 515fd709e74c57ba4fc1b7dac208d8df8492ba06 Mon Sep 17 00:00:00 2001
From: ghost <andreas.monzner@dream-property.net>
Date: Tue, 14 Jul 2020 16:41:18 +0200
Subject: [PATCH] NimManager: handle unicable.xml entries with "UniSEqC" as
 EN50494 (Unicable1) again unicables.xml: changed SUS 41 FX to JESS

---
 data/unicable.xml                   | 4 ++--
 lib/python/Components/NimManager.py | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/usr/share/enigma2/unicable.xml b/usr/share/enigma2/unicable.xml
index fb6d156..3ecd641 100644
--- a/usr/share/enigma2/unicable.xml
+++ b/usr/share/enigma2/unicable.xml
@@ -401,8 +401,8 @@ unicable (version)
 			<product name="SUS 21 F" format="EN50494" positions="2" scr1="1076" scr2="1178"/>
 			<product name="SUS 21 FI" format="EN50494" positions="2" scr1="1210" scr2="1420"/>
 			<product name="SUS 4441 F" format="EN50494" scr1="1210" scr2="1420" scr3="1680" scr4="2040"/>
-			<product name="SUS 41 FX" format="UniSEqC" positions="4" scr1="1076" scr2="1178" scr3="1280" scr4="1382"/>
-			<product name="SUS 5541 F-NFA" format="EN 50494" scr1="0974" scr2="1076" scr3="1178" scr4="1280"/>
+			<product name="SUS 41 FX" format="JESS" positions="4" scr1="1076" scr2="1178" scr3="1280" scr4="1382"/>
+			<product name="SUS 5541 F-NFA" format="UniSEqC" scr1="0974" scr2="1076" scr3="1178" scr4="1280"/>
 			<product name="SUS 5581 F-NFA" format="UniSEqC" scr1="0974" scr2="1076" scr3="1178" scr4="1280" scr5="1382" scr6="1484" scr7="1586" scr8="1688"/>
 			<product name="SUS 5581 NF 1x8" format="UniSEqC" scr1="1068" scr2="1284" scr3="1400" scr4="1516" scr5="1632" scr6="1748" scr7="1864" scr8="1980"/>
 			<product name="SUS 5581 NF 3x3 1" format="UniSEqC" scr1="1068" scr2="1284" scr3="1400"/>
diff --git a/usr/lib/enigma2/python/Components/NimManager.py b/usr/lib/enigma2/python/Components/NimManager.py
index 4ee1af8..bdb2370 100644
--- a/usr/lib/enigma2/python/Components/NimManager.py
+++ b/usr/lib/enigma2/python/Components/NimManager.py
@@ -1583,7 +1583,7 @@ class UnicableProducts(object):
 				scr=[]
 				lscr = []
 				format = product.get("format", "EN50494").upper()
-				if format in ('JESS', 'UNICABLE2', 'SCD2', 'EN50607', 'EN 50607', 'UNISEQC'):
+				if format in ('JESS', 'UNICABLE2', 'SCD2', 'EN50607', 'EN 50607'):
 					format = "EN50607"
 				else:
 					format = "EN50494"
-- 
2.1.4

